azdev/operations/style.py

```diff
@@ -74,32 +74,27 @@ def check_style(modules=None, pylint=False, pep8=False, git_source=None, git_tar
         pylint = True
 
     exit_code_sum = 0
-    if pep8:
-        pep8_result = _run_pep8(selected_modules)
-        exit_code_sum += pep8_result.exit_code
 
     if pylint:
         pylint_result = run_pylint(selected_modules)
         exit_code_sum += pylint_result.exit_code
 
-    display('')
-    subheading('Results')
-
-    # print success messages first
-    if pep8_result and not pep8_result.error:
-        display('Flake8: PASSED')
-    if pylint_result and not pylint_result.error:
-        display('Pylint: PASSED')
+        if pylint_result.error:
+            display('Pylint: PASSED')
+            logger.error(pylint_result.error.output.decode('utf-8'))
+            logger.error('Pylint: FAILED\n')
+        else:
+            display('Pylint: PASSED\n')
```

Comment:
Adjust the code sequence to let `pylint` errors be shown as soon as possible, instead of after the execution of both `pylint` and `flake8`.